jplag/src/main/java/de/jplag/Submission.java

```diff
@@ -147,10 +149,10 @@ public int getSimilarityDivisor(boolean subtractBaseCode) {
     }
```

Comment:
This is not part of your PR; but getSimilarityDivisor() should be package-private. There is no reason for it to be part of the public API.